distdens/fillout.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def fillout(x, y, limits=None, **kwargs):
+    if limits is None:
+        limits = [min(x), max(x), min(y), max(y)]
+    x_inicial = np.array(limits[0]).reshape(1, 1)
+    x_final = np.array(limits[1]).reshape(1, 1)
+    y_inicial = np.array(limits[2]).reshape(1, 1)
+    y_final = np.array(limits[3]).reshape(1, 1)
+    indice_pegado_costa = np.where(x == min(x))[0][0]
+    closed_x = _close_coordinate(x, indice_pegado_costa)
+    closed_y = _close_coordinate(y, indice_pegado_costa)
+    x_prueba = np.concatenate(
+        [
+            x_inicial,
+            x_inicial,
+            x_final,
+            x_final,
+            x_inicial,
+            np.flip(closed_x.reshape(len(closed_x), 1)),
+        ]
+    )
+    y_prueba = np.concatenate(
```

Comment:
Por favor refactoriza las líneas 10-11 y 25-34 para que la 25 quedara algo así:
```suggestion
    y_prueba = _testing(limits[2:4], closed_y)
```